hermes/file_gen_test.go

```diff
@@ -25,18 +28,18 @@ type FileNameTestUnit struct {
 	want     string
 }
 
-var test_unit_table [5]FileNameTestUnit
+var testUnitTable [5]FileNameTestUnit
 
-// Function completes the table of HermesFileTestUnit by iterating over its entries
+// completeTestUnitTable completes the table of HermesFileTestUnit by iterating over its entries
```

Comment:
Avoid logic in test code. This should be a static slice instead, declared within the test function that uses it.
In general: move the declarations closest to their use.

Reply:
@alicjakwie is making these changes in a different PR: https://github.com/googleinterns/step224-2020/pull/58 so I'll mark this resolved for now.